MobileWallet/Info.plist

```diff
@@ -126,5 +126,17 @@
 	<string>Light</string>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<true/>
+    <key>NSUbiquitousContainers</key>
+    <dict>
+        <key>iCloud.com.tari.wallet</key>
+        <dict>
+            <key>NSUbiquitousContainerIsDocumentScopePublic</key>
+            <true/>
+            <key>NSUbiquitousContainerName</key>
+            <string>Tari Wallet</string>
```

Comment:
`Tari Aurora` may be a better choice here and everywhere in general.

Reply:
This is Jay choice. And only admin can change it

Reply:
Ok.